src/js/view/controls/components/simple-tooltip.js

```diff
@@ -19,6 +19,10 @@ export function SimpleTooltip(attachToElement, name, text, openCallback, closeCa
             if (instance.touchEvent) {
                 return;
             }
+            if (instance.suppress) {
+                instance.suppress = false;
```

Comment:
what implications does this have for other menus using this option?

Reply:
Currently, this is only used for menus, but I can see potential for other use cases.